web_company_color/models/res_company.py

```diff
@@ -73,7 +124,32 @@ def write(self, values):
                 "color_navbar_bg",
                 "color_navbar_bg_hover",
                 "color_navbar_text",
+                "color_button_bg",
+                "color_button_bg_hover",
+                "color_button_text",
+                "color_link_text",
+                "color_link_text_hover",
             )
+            if "logo" in values:
```

Comment:
You seem to be undoing https://github.com/OCA/web/pull/2158/files. Is that on purpose?

Reply:
Ah, good catch. I didn't really follow this module's history and just ported the commit trying to preserve as much of it as possible.

Thank you :pray: 
Fixing it now.